time_entry/models.py

```diff
@@ -3,6 +3,10 @@
 # Create your models here.
 class TimeEntry(models.Model):
     date=models.DateField(default=datetime.now)
+    #test update
+    punchInTime = models.TimeField(default=datetime.now)
+    punchOutTime = models.TimeField(default=datetime.now)
+    #test update end
```

Comment:
about the punchIn punchOut time, i feel like we don't have enough data to handle this. since for some companies, they may have punchin and punchout time,  but for others, they simply record how many hours an employee worked for that day. so I only keep track of the hoursWorked in the db.